game/screens/menu.py

```
@@ -130,8 +150,8 @@ def draw(self):
                 (self.music_rect,  "MUSIC"),
                 (self.about_rect,  "HOW TO PLAY"),
             ]
-            if self.switch_rect is not None and self._switch_label:
-                items.append((self.switch_rect, self._switch_label))
+            for rect, (_, label) in zip(self.switch_rects, self._switch_targets):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)